Composer/packages/client/src/utils/electronUtil.ts

```diff
@@ -5,5 +5,5 @@
  * Returns true if the client is embedded in the Composer Electron environment.
  */
 export function isElectron(): boolean {
-  return !!window.__IS_ELECTRON__;
+  return window.__IS_ELECTRON__ ?? false;
```

Comment:
```suggestion
  return Boolean(window.__IS_ELECTRON__);
```

It could be possible for `window.__IS_ELECTRON__` to be something other than a boolean.